doc/metadata/README.md

```diff
@@ -0,0 +1,36 @@
+Verify in-toto's supply chain with in-toto
+===========================================
+
+**Overview**  
+This document describes how to use in-toto to verify the software supply chains for the PyPi source of in-toto. In this documentation, We are using in-toto v1.0.1 as an example.
+
+The basic supply chains are laid out in "in_toto_sdist.layout" and consist of two steps:
+
+1. "git-tag" -- Tag the release.
+2. "create-sdist" -- Creates source distribution (.tar.gz) using the same materials from (1).
+
+To create link files, we may want to start with a clean clone or using ```--exclude``` to make sure that we don't record any additional files (such as egg files and pyc files) as materials.
+
+We will use the following commands to create the link files:
+
+1. ```in-toto-run --step-name git-tag --materials . --key owner -- git tag --sign v1.0.1 -m "v1.0.1"```
+2. ```in-toto-run --step-name create-sdist --materials . --products in-toto.1.0.1.tar.gz --key owner -- python setup.py sdist```
+
+The example links are laid out in "git-tag.776a00e2.link" and "create-sdist.776a00e2.link". The key for signing the steps is the bob's key from the in-toto demo: https://github.com/in-toto/demo/tree/master/functionary_bob
+
+To verify the supply chain, we can run the in-toto verify command:  
+```in-toto-verify -l in-toto-sdist-signed.layout -k keys/alice.pub -v```
```

Comment:
The inspection here will fail. We should include the current .tar.gz for the sake of testing, and maybe update that to the source of v1.1.0 when we're ready to release, using that as an example moving forward. The example throughout this guide can be for v1.1.0, I suspect.

Reply:
@SolidifiedRay and I verified that the inspection works with the source archive present, so essentially we have a working layout for this minimal supply chain!

Reply:
We did need to update the untar command though, it used a CLI argument to pass in the source archive. We switched to regular `tar` as we do in in-toto/demo.